src/Dianoga/Optimizers/CommandLineToolOptimizer.cs

```diff
@@ -38,6 +38,7 @@ public virtual string AdditionalToolArguments
 			}
 		}
 
+		public bool DisableResizing { get; set; }

```

Comment:
The idea was to allow other processors to do the same, but since we haven't implemented it in any other, you are right, I have moved it to the WEbPOptmizer